earth_enterprise/src/fusion/gepublish/geserveradmin.cpp

```diff
@@ -93,6 +93,10 @@ void usage(const std::string &progn, const char *msg = 0, ...) {
     "   [--vhname <vh_name>]     If virtual host is omitted,\n"
     "                            <db_name> is published to the default\n"
     "                            virtual host: \"%s\".\n"
+    "   [--setecdefault]          Publish this database as the default one for\n"
```

Comment:
Lint: Now this help text is indented one space too much. :stuck_out_tongue:

Reply:
Ooop!